fipvlan.c

```diff
@@ -624,8 +624,16 @@ create_and_start_vlan(struct fcf *fcf, bool vn2vn)
 				    real_dev->ifname, fcf->vlan, vlan->ifname);
 			rc = 0;
 		} else {
-			snprintf(vlan_name, IFNAMSIZ, "%s.%d%s",
-				 real_dev->ifname, fcf->vlan, config.suffix);
+			rc = snprintf(vlan_name, IFNAMSIZ, "%s.%d%s",
+					real_dev->ifname, fcf->vlan,
+					config.suffix);
+			if (rc >= IFNAMSIZ) {
```

Comment:
Shouldn't we also check for (rc < 0) here?

Reply:
You're right, I'll update that.